extensions/amp-social-share/1.0/social-share.js

```diff
@@ -157,6 +173,20 @@ function createEndpoint(typeConfig, baseEndpoint, props) {
   return finalEndpoint;
 }
 
+/**
+ * @param {!JsonObject} props
+ * @return {!JsonObject}
+ */
+function unpackParams(props) {
```

Comment:
It looks like you are essentially using the `...rest` pattern instead of defining `unpackParams`. We do something similar in amp-selector if you want to do that here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-selector/1.0/selector.js#L28-L37

Reply:
Thanks, updated to use this pattern!